joomla/joomla-cms
# Ensure a language is set in all views

### Summary of Changes
Follow up pr for #40614 which stabilizes the use of the internal language object.

### Testing Instructions
- Turn on debug mode in the global configuration without it, you wont see an error
- Log in on the front end
- Open the url /index.php?option=com_users&view=reset

### Actual result BEFORE applying this Pull Request
Error that language is not set.

### Expected result AFTER applying this Pull Request
Profile page is loading.

### Link to documentations
Please select:
